src/main/java/custom_exceptions/BookHandlingExceptions.java

```diff
@@ -0,0 +1,8 @@
+package custom_exceptions;
+
+public class BookHandlingExceptions extends Exception {
```

Comment:
`BookHandlingExceptions` can be a packagename and specific exceptions can be added to the package.